src/messages/mod.rs

```diff
@@ -123,6 +124,7 @@ impl Message {
     /// Creates a signed message from single node.
     pub(crate) fn single_src(
         keypair: &Keypair,
+        age: u8,
```

Comment:
Should this not take &Node instead of KeyPair + age?

Reply:
keypair is from self, and `age` is fetched from SectionPeers info.
I thought Node won't contain the info of age ?